data/transactions/logic/eval.go

```diff
@@ -262,14 +267,14 @@ func EvalStateful(program []byte, params EvalParams) (pass bool, delta basics.Ev
 
 	cx.appEvalDelta = basics.EvalDelta{
 		GlobalDelta: make(basics.StateDelta),
-		LocalDeltas: make(map[basics.Address]basics.StateDelta),
+		LocalDeltas: make(map[uint64]basics.StateDelta),
```

Comment:
seems like can be preallocated to len(txn.Accounts)

Reply:
updated -- did `len(params.Txn.Txn.Accounts)+1` since sender is there too